apps/api/src/modules/agents/service.ts

```
@@ -1,71 +1,54 @@
-import { and, asc, eq, isNull } from "drizzle-orm";
+import { asc, eq } from "drizzle-orm";
```

ESLint

[error] Resolve error: EACCES: permission denied, open '/LrRnMcutib'
    at Object.writeFileSync (node:fs:2409:20)
    at l (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:13685)
    at createFilesMatcher (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:14437)
    at Object.resolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import@2.32.0_eslint@9.33.0_jiti@2.5.1_/node_modules/eslint-import-resolver-typescript/lib/index.cjs:298:107)
    at withResolver (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.40.0_eslint@9.33.0_jiti@2.5.1__t_9d7974006af4c22e76766645224f0f11/node_modules/eslint-module-utils/resolve.js:180:23)
    at fullResolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.40.0_eslint@9.33.0_jiti@2.5.1__t_9d7974006af4c22e76766645224f0f11/node_modules/eslint-module-utils/resolve.js:201:22)
    at relative (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.40.0_eslint@9.33.0_jiti@2.5.1__t_9d7974006af4c22e76766645224f0f11/node_modules/eslint-module-utils/resolve.js:217:10)
    at resolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.40.0_eslint@9.33.0_jiti@2.5.1__t_9d7974006af4c22e76766645224f0f11/node_modules/eslint-module-utils/resolve.js:233:12)
    at ExportMapBuilder.get (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.40.0_eslint@9.33.0_jiti@2.5.1___292a96fbf5bd2ee1b050c63b51d0dca8/node_modules/eslint-plugin-import/lib/exportMap/builder.js:37:45)
    at processBodyStatement (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.40.0_eslint@9.33.0_jiti@2.5.1___292a96fbf5bd2ee1b050c63b51d0dca8/node_modules/eslint-plugin-import/lib/rules/namespace.js:12:38)

(import/namespace)